docs/overview.md

```diff
@@ -0,0 +1,70 @@
+# Overview
+
+## Goals
+Help infrastructure platform teams build self-service developer platforms that work consistently across clouds. We are doing this by creating a K8s-native way of orchestrating cloud services and kubernetes resources.
+
+
+## Customer Context
+we have found that our customers often use  ACK/ASO/KCC as an input into a custom-built internal developer platform (IDP). In nearly all cases, platform teams are maintaining a custom orchestration layer between the developer interface and the cloud resources. The orchestration layer is typically responsible for: 
+* RBAC
+* Resource creation ordering
+* Continuous reconciliation
+* Hydration of templates with developer inputs
+* Security checks
+* Automated and manual approval workflows
+* Abstracting platform complexity for developer ease-of-use 
+
+There is no consistent K8s-native set of tooling used to build this orchestration layer. The most common solution is to write a layered set of Helm charts, with the top layer exposing only the parameters available to developers. Existing solutions tend to be challenging to maintain and require additional investment in hydration pipelines and custom RBAC models. Customers have been asking us to help address the challenges of orchestrating across different cloud services and teams / roles. The solution must be flexible/extensible so that existing solutions can be “plugged in”, but relatively simple to use out-of-the box in a greenfield scenario. Additionally, our customers have been asking for a consistent approach to cloud resource orchestration across cloud service vendors.
```

Comment:
@nicslatts - Do you have thoughts on the this comment ?

Reply:
probably it's more accurate to say they are more challenging to scale and troubleshoot. Troubleshooting a values.helm file can be a nightmare, as well as detecting whether values are even being honored at all, and other solutions like Terraform are harder to scale because they are originally intended for human usage rather than automated pipelines.